<ui version="4.0" >
 <author>Peter Hedlund &lt;peter.hedlund@kdemail.net></author>
 <class>MCQueryDlgForm</class>
 <widget class="QWidget" name="MCQueryDlgForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>450</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" >
   <item>
    <widget class="QGraphicsView" name="imageGraphicsView" >
     <property name="minimumSize" >
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QLabel" name="TextLabel1" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font" >
        <font>
         <family>Tahoma</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text" >
        <string>Select the correct translation:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="orgField" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="transgroup" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string/>
       </property>
       <layout class="QGridLayout" >
        <property name="leftMargin" >
         <number>8</number>
        </property>
        <property name="topMargin" >
         <number>8</number>
        </property>
        <property name="rightMargin" >
         <number>8</number>
        </property>
        <property name="bottomMargin" >
         <number>8</number>
        </property>
        <property name="horizontalSpacing" >
         <number>6</number>
        </property>
        <property name="verticalSpacing" >
         <number>6</number>
        </property>
        <item row="4" column="1" >
         <widget class="QLabel" name="trans5" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>-</string>
          </property>
          <property name="wordWrap" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0" >
         <widget class="QRadioButton" name="rb_trans5" >
          <property name="text" >
           <string>&amp;5:</string>
          </property>
          <property name="shortcut" >
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QLabel" name="trans4" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>-</string>
          </property>
          <property name="wordWrap" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QRadioButton" name="rb_trans4" >
          <property name="text" >
           <string>&amp;4:</string>
          </property>
          <property name="shortcut" >
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLabel" name="trans3" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>-</string>
          </property>
          <property name="wordWrap" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QRadioButton" name="rb_trans3" >
          <property name="text" >
           <string>&amp;3:</string>
          </property>
          <property name="shortcut" >
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLabel" name="trans2" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>-</string>
          </property>
          <property name="wordWrap" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QRadioButton" name="rb_trans2" >
          <property name="text" >
           <string>&amp;2:</string>
          </property>
          <property name="shortcut" >
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLabel" name="trans1" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>-</string>
          </property>
          <property name="wordWrap" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QRadioButton" name="rb_trans1" >
          <property name="text" >
           <string>&amp;1:</string>
          </property>
          <property name="shortcut" >
           <string>1</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <property name="horizontalSpacing" >
        <number>6</number>
       </property>
       <property name="verticalSpacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <layout class="QGridLayout" >
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <property name="horizontalSpacing" >
          <number>6</number>
         </property>
         <property name="verticalSpacing" >
          <number>6</number>
         </property>
         <item row="1" column="1" >
          <widget class="QPushButton" name="dont_know" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Skip (Do &amp;Not Know)</string>
           </property>
          </widget>
         </item>
         <item rowspan="2" row="0" column="0" >
          <widget class="QPushButton" name="show_all" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Show &amp;Solution</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QPushButton" name="know_it" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Skip (I &amp;Know It)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item rowspan="2" row="0" column="1" >
        <widget class="QGroupBox" name="progressgroup" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Progress</string>
         </property>
         <layout class="QGridLayout" >
          <property name="leftMargin" >
           <number>11</number>
          </property>
          <property name="topMargin" >
           <number>11</number>
          </property>
          <property name="rightMargin" >
           <number>11</number>
          </property>
          <property name="bottomMargin" >
           <number>11</number>
          </property>
          <property name="horizontalSpacing" >
           <number>6</number>
          </property>
          <property name="verticalSpacing" >
           <number>6</number>
          </property>
          <item row="0" column="1" >
           <widget class="QProgressBar" name="countbar" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QProgressBar" name="timebar" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="countlabel" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Count:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="timelabel" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Time:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="Cyclelabel" >
            <property name="frameShape" >
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text" >
             <string>Cycle:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QLabel" name="progCount" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="status" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>111</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="stopPracticeButton" >
         <property name="text" >
          <string>&amp;Stop Practice</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editEntryButton" >
         <property name="text" >
          <string>&amp;Edit Entry...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>rb_trans1</tabstop>
  <tabstop>rb_trans2</tabstop>
  <tabstop>rb_trans3</tabstop>
  <tabstop>rb_trans4</tabstop>
  <tabstop>show_all</tabstop>
  <tabstop>know_it</tabstop>
  <tabstop>dont_know</tabstop>
  <tabstop>rb_trans5</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
